src/main/java/it/lucchesi/tools/DateTools/DateTools.java

```diff
@@ -26,6 +26,8 @@ public static long weekdays(LocalDate start, LocalDate end) {
 		Preconditions.checkArgument(Preconditions.checkNotNull(start)
 				.compareTo(Preconditions.checkNotNull(end)) <= 0);
 		
+		if(start.equals(end)) return 1;
```

Comment:
It doesn't work when the same date is Saturday or Sunday, in this case it should return 0.